tasks/broad/Dragen.wdl

```diff
@@ -0,0 +1,113 @@
+version 1.0
+
+## Copyright Broad Institute, 2020
+##
+## This WDL defines tasks to use Dragen's DRAGstr approach to STR sequencing artifacts 
+## Indel genotype priors in the DRAGEN-Gatk pipeline. 
+##
+## Runtime parameters are often optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+
+task ComposeSTRTableFile {
```

Comment:
This appears to be based solely on the reference files (rather than unique input files). Is this something we should do every time we run the pipeline, or could we consider generating this file as a reference file that is passed in to the pipeline as an input?

Reply:
This is done once per reference, and we're planning on uploading it to the public references bucket. Should we include this task in the WDL then or not?

Reply:
If the files are available in the public bucket, I think it would make more sense to include them as input reference files rather than regenerate them each time. If we think that users may want to use non-standard references we could either provide documentation for how to generate these files or we could include the input files as optional and only run this task if the input is not provided

Reply:
Agreed, it should not be in the WDL if it is generated once. You can pull the code to generate it out into a separate script or documentation though, if you think we'll want to regenerate it occasionally.

Reply:
Ok, I removed the task from the WDL. The STR table file is already passed as an (optional) input file, so no other changes are required.